packages/fxa-settings/src/components/PageRecoveryKeyAdd/index.test.tsx

```diff
@@ -131,6 +131,13 @@ describe('PageRecoveryKeyAdd', () => {
         account.createRecoveryKey = jest
           .fn()
           .mockResolvedValue(new Uint8Array(20));
+      } else {
+        account.createRecoveryKey = jest
+          .fn()
+          .mockImplementation(() => {
+            // this is just a random key that does match a real error to keep tests passing.
```

Comment:
Oh, does TS not complain here? I did this first trying to mock our error and thought I saw red squigglies, but if those are here we can just add a `@ts-ignore` and that's fine by me.

Reply:
Oddly enough, no -- I saw yellow squigglies on this but then it didn't throw in CI. I wonder if it's a test thing.

Reply:
Oh interesting no, it was just eslint.   Nothing from ts!

Reply:
double interesting. I thought ESLint would have flagged this as a [`no-throw-literal`](https://eslint.org/docs/latest/rules/no-throw-literal) violation.
Although I guess that isn't in the `eslint:recommended` config (per https://eslint.org/docs/latest/rules/), or presumably in any of the https://github.com/mozilla/eslint-plugin-fxa/search?q=literal&type=code configs.

Reply:
Oh yeah, it's that eslint rule that Peter called out.

Is eslint set up in VSCode for you or is it mine set up incorrectly? 😅

![image](https://user-images.githubusercontent.com/13018240/184022692-180a6236-8c71-437e-8aa8-7fd0e1b1abae.png)

Reply:
I got the eslint error in my IDE, but tbh since we're doing it deliberately and it's in a test (and not causing a problem in CI), I think it's clearer to just leave it as is?

Reply:
I'm fine leaving it as-is because I agree, but I still think it might be nice to disable eslint for that line since it's intentional, otherwise the IDE will point to that file as one having a "problem". I can't remember if it changes the color of the file in the tree (I'm in another branch currently) but I think it does and I know if you hover over the file it'll say "1 problem in this file").